client/src/components/BulkUpdateForm/BulkUpdateForm.js

```diff
@@ -129,7 +129,7 @@ export default class BulkUpdateForm extends React.Component{
 
             for (let i = 0; i < fileUpload.images.length; i++) {
                 const image = fileUpload.images[i]
-                if (!image.name.endsWith('.png') && !image.name.endsWith('.jpg')){
+                if (!image.name.endsWith('.png') && !image.name.endsWith('.jpg') && !image.name.endsWith('.PNG') && !image.name.endsWith('.JPG') && !image.name.endsWith('.jpeg') && !image.name.endsWith('.JPEG')){
```

Comment:
Could you use something like toLowerCase() for this check rather than typing out all extensions in both cases?

Reply:
no, because we don't want to allow stuff like `pNg` or `JPeg`